parsers/CA_BC.py

```diff
@@ -1,17 +1,71 @@
 #!/usr/bin/env python3
 
 # The arrow library is used to handle datetimes
-from datetime import datetime
+from datetime import datetime, timedelta
 from logging import Logger, getLogger
 from typing import Optional
 
 import arrow
-from requests import Session
+import pandas as pd
+import pytz
+from requests import Response, Session
+
+from electricitymap.contrib.config import ZONES_CONFIG, ZoneKey
+from electricitymap.contrib.lib.models.event_lists import TotalConsumptionList
+from electricitymap.contrib.parsers.lib.config import refetch_frequency
+from electricitymap.contrib.parsers.lib.exceptions import ParserException
```

Comment:
```suggestion
from parsers.lib.config import refetch_frequency
from parsers.lib.exceptions import ParserException
```
For windows again, I think we should try and move these from parsers lib to contrib lib now that we have the other models and such as well in there.